packages/application-generic/src/usecases/get-subscriber-template-preference/get-subscriber-template-preference.usecase.ts

```diff
@@ -253,6 +254,7 @@ function mapTemplateConfiguration(
     name: template.name,
     tags: template?.tags || [],
     critical: template.critical != null ? template.critical : true,
+    triggers: template.triggers,
```

Comment:
can you please update the Swagger API response in the subscriber's controller?

Reply:
Definitely!  i knew I forgot something here 🤦‍♂️